062-worker-pool/proposal.md

```diff
@@ -0,0 +1,157 @@
+# Summary
+
+This proposal outlines the support for worker pool. A worker pool is a group
+of Concourse workers, that can be associated with one or multiple teams.
```

Comment:
For what it's worth, when I originally heard of "worker pools", I was thinking more along the lines of allowing different Concourse installations to coordinate with each other to access a common pool of workers, which would make it much easier to launch new Concourse installations for different teams by giving them access to some workers already by default and not have to set up workers separately in order to get started.